### PR TITLE
Functional tests for `<bidder>.usersync.enabled` property

### PR DESCRIPTION
**Pull request by marki1an**

### CODE REVIEW

File: src/test/groovy/org/prebid/server/functional/tests/UserSyncSpec.groovy [hunk 1]
```diff
@@ -13,6 +13,9 @@ import static org.prebid.server.functional.testcontainers.Dependencies.getNetwor
 
 class UserSyncSpec extends BaseSpec {
 
+    private static final Map GENERIC_USERSYNC_CONFIG = ["adapters.${GENERIC.value}.usersync.${IFRAME.value}.url"         : "$networkServiceContainer.rootUri/generic-usersync&redir={{redirect_url}}".toString(),
```
Comment (Net-burst) [inline src/test/groovy/org/prebid/server/functional/tests/UserSyncSpec.groovy:16]: Please don't use raw types. Either use `def` or use fully-qualified types.

File: src/test/groovy/org/prebid/server/functional/tests/UserSyncSpec.groovy [hunk 2]
```diff
@@ -132,4 +135,51 @@ class UserSyncSpec extends BaseSpec {
...
+
+        when: "PBS processes cookie sync request without cookies"
+        def response = prebidServerService.sendCookieSyncRequest(cookieSyncRequest)
+
+        then: "Response should contain synced bidder"
+        assert response.getBidderUserSync(GENERIC)
+        assert response.bidderStatus.size() == 1
+    }
+
+    def "PBS cookie sync should sync bidder which bidder.usersync.enabled=true"() {
```
Comment (Net-burst) [inline src/test/groovy/org/prebid/server/functional/tests/UserSyncSpec.groovy:154]: "PBS cookie sync should sync bidder when bidder.usersync.enabled=true"

File: src/test/groovy/org/prebid/server/functional/tests/UserSyncSpec.groovy [hunk 3]
```diff
@@ -132,4 +135,51 @@ class UserSyncSpec extends BaseSpec {
...
+
+        when: "PBS processes cookie sync request without cookies"
+        def response = prebidServerService.sendCookieSyncRequest(cookieSyncRequest)
+
+        then: "Response should contain synced bidder"
+        assert response.getBidderUserSync(GENERIC)
+        assert response.bidderStatus.size() == 1
+    }
+
+    def "PBS cookie sync shouldn't sync bidder which bidder.usersync.enabled=false"() {
```
Comment (Net-burst) [inline src/test/groovy/org/prebid/server/functional/tests/UserSyncSpec.groovy:170]: "PBS cookie sync shouldn't sync bidder when bidder.usersync.enabled=false"

**Review by Net-burst — CHANGES_REQUESTED** [review]

**Review by SerhiiNahornyi — APPROVED** [review]

**Review by Net-burst — COMMENTED** [review]
LGTM

**Pull request merged**